db.json

```diff
@@ -2231,7 +2231,7 @@
         "args": [
             "{{prefix}}"
         ],
-        "beamline": "TMO",
+        "beamline": "",
```

Comment:
Does anyone know why the TMO PPMs were made not-TMO?

Reply:
That's a strange one, and it makes me wonder if we should write a wrapper to happi commands that updates a log indicating who changed the device config and when. Though this looks like it may have been a direct edit...

Reply:
Agreed. And yes, this was a direct edit because the last update time is not changed.